tests/test_counting_hash.py

```diff
@@ -35,6 +38,10 @@ def teardown():
     utils.cleanup()
 
 
+def scriptpath(script):
+    return script
```

Comment:
Without it,  I get this error:  " global name 'scriptpath' is not defined", for this line:
script = scriptpath('load-into-counting.py')
 your recommendation is welcomed?